### ISSUE TITLE
Error: "the condition has length > 1" when using metaPR2 shiny R package

### ISSUE THREAD

**Issue by mjamy** [pr2database/metapr2-shiny#2]
Hi Daniel, 

I wanted to use the metaPR2 shiny R package as the metaPR2 website seemed to be down. 

I installed the package from the R-universe and launched the shiny interface with `metapr2::run_app()`. However, I get the following error (please see screenshot). Any idea why this could be?

Here is the `sessionInfo`:

```
R version 4.2.2 (2022-10-31)
Platform: aarch64-apple-darwin20 (64-bit)
Running under: macOS Ventura 13.0.1

Matrix products: default
LAPACK: /Library/Frameworks/R.framework/Versions/4.2-arm64/Resources/lib/libRlapack.dylib

locale:
[1] en_US.UTF-8/en_US.UTF-8/en_US.UTF-8/C/en_US.UTF-8/en_US.UTF-8

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base     

other attached packages:
[1] shiny_1.7.4   metapr2_2.0.0
```

Many thanks!
Mahwash

<img width="1433" alt="image" src="https://user-images.githubusercontent.com/40695699/219422645-95e02b52-ceec-4a56-a73c-7619f906c605.png">

**Comment by vaulot** [comment]
Thanks Mawash

This error was caused by a change between R 4.2 and previous version in the way R was treating  `if``.
This has now been fixed (new version 2.0.1 of the R package).

**Comment by mjamy** [comment]
Wonderful thank you Daniel! Just checked and it runs perfectly without the error :)
